### PR TITLE
[Playground] Add Token components, fix links

### PR DESCRIPTION
**Pull request by kien-ngo**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
This PR introduces new components and updates documentation related to token management in a web application. It enhances the user interface for displaying token information and adds a new token page.

### Detailed summary
- Added a new token entry in `navLinks.ts`.
- Updated `docsLink` in `page.tsx` files for better documentation.
- Introduced a new `Token` page with components for displaying token details.
- Added SVG representation for USDC in `public/usdc.svg`.
- Implemented various token components in `token-examples.tsx`.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by kien-ngo** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5677?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#5677** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5677?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5677?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#5657** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5657?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: apps/playground-web/src/components/headless-ui/token-examples.tsx [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+      <CodeExample
+        preview={
+          <TokenProvider
+            address={USDC_ADDRESS}
+            client={THIRDWEB_CLIENT}
+            chain={ethereum}
+          >
+            <TokenIcon
+              className="h-auto w-20 rounded-full"
+              iconResolver="/usdc.svg"
```
Comment (joaquim-verges) [inline apps/playground-web/src/components/headless-ui/token-examples.tsx:79]: is this the only way? what happens if i pass nothing?

i think we have all these icons in the pay service, could we just fetch those from there for tokens we know about?

Comment (kien-ngo) [reply apps/playground-web/src/components/headless-ui/token-examples.tsx:79]: Oh I did not know we have the icons over there. I will check an update it in a follow up PR

File: apps/playground-web/src/components/headless-ui/token-examples.tsx [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+  return (
+    <TokenProvider
+      address={NATIVE_TOKEN_ADDRESS}
+      client={THIRDWEB_CLIENT}
+      chain={ethereum}
+    >
+      <TokenName loadingComponent={<span>Loading...</span>} />
+    </TokenProvider>
+  )
+}`}
```
[the remark this answers is not in the document]
Comment (kien-ngo) [reply apps/playground-web/src/components/headless-ui/token-examples.tsx:138]: acknowleded

File: apps/playground-web/src/components/headless-ui/token-examples.tsx (hunk 1 above)
Comment (kien-ngo) [reply apps/playground-web/src/components/headless-ui/token-examples.tsx:79]: https://thirdwebdev.slack.com/archives/C04LA5X7FNE/p1733916180045149

**Review by joaquim-verges — APPROVED** [review]

**Pull request merged**